File: 资料库/selenium/Selenium_day2_上午/Selenium_day2_上午/lintyleo-WeekendSelenium-master/WeekendSelenium-master/JavaSeleniumDemo/src/main/java/com/hello/RanzhiTestCases.java
package com.hello;

import org.junit.Assert;
import org.junit.Test;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

/**
 * Created by Linty on 9/4/2016.
 */
public class RanzhiTestCases {

    @Test
    public void testLogIn() {
        WebDriver driver = new FirefoxDriver();
        String baseUrl = "http://localhost:808/ranzhi/www";

        driver.get(baseUrl);
        try {
            Thread.sleep(2000);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }

        driver.findElement(By.className("btn")).click();
        driver.findElement(By.linkText("简体")).click();
        try {
            Thread.sleep(1000);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }

        WebElement account = driver.findElement(By.id("account"));
        account.clear();
        account.sendKeys("admin");

        WebElement password = driver.findElement(By.id("password"));
        password.clear();
        password.sendKeys("admin");

        driver.findElement(By.id("submit")).click();

        try {
            Thread.sleep(2000);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }

        String expected = baseUrl + "/sys/index.html";
        String actual = driver.getCurrentUrl();
        Assert.assertEquals(expected, actual);

        driver.quit();

    }
}

File: 资料库/selenium/Selenium_day2_上午/Selenium_day2_上午/lintyleo-WeekendSelenium-master/WeekendSelenium-master/PythonDemo/ranzhi_testcases.py
# 引入unittest模组
import unittest
from time import sleep

from selenium import webdriver


# 继承unittest.TestCase基类
class RanzhiTestCases(unittest.TestCase):
    # define a method
    # 测试方法以test开头
    def test_log_in(self):
        base_url = "http://localhost:808/ranzhi/www"
        driver = webdriver.Firefox()
        driver.get(base_url)
        sleep(2)

        # 改变语言为中文简体
        driver.find_element_by_class_name("btn").click()
        driver.find_element_by_link_text("简体").click()
        sleep(1)

        account = driver.find_element_by_id("account")
        account.clear()
        account.send_keys("admin")

        password = driver.find_element_by_id("password")
        password.clear()
        password.send_keys("admin")

        driver.find_element_by_id("submit").click()
        sleep(2)

        expected = base_url + "/sys/index.html"
        actual = driver.current_url
        # 断言
        self.assertEqual(expected, actual, "登录页面导航失败")

        driver.quit()
